third_party/infineon/trustm/BUILD.gn

```diff
@@ -1,76 +1,73 @@
-# Copyright (c) 2020 Project CHIP Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
Copyright date as well. Please check all of them.

Reply:
Hi @tcarmelveilleux Thanks for your comments. We already fixed this in the latest commit.